### ISSUE TITLE
Possible wrong code

### ISSUE THREAD

**Issue by YichiZhang0613** [GrayJack/janetrs#12]
In janetrs-dev/src/types/array.rs, maybe in code, " if index < 0 || index > self.len() + 1" should be " if index < 0 || index > self.len()" ?
```rust
/// # Janet Panics
    /// Janet panics if `index < 0` or `index > len`.
    ///
    /// # Examples
    /// ```
    /// use janetrs::array;
    /// # let _client = janetrs::client::JanetClient::init().unwrap();
    ///
    /// let mut array = array![1, 2];
    /// array.insert(1, 3) // now it's `[1, 3, 2]`
    /// ```
    #[cfg_attr(feature = "inline-more", inline)]
    pub fn insert(&mut self, index: i32, element: impl Into<Janet>) {
        if index < 0 || index > self.len() + 1 {
            crate::jpanic!(
                "insertion index (is {}) should be >= 0 and <= {})",
                index,
                self.len()
            );
```

**Comment by GrayJack** [comment]
Yes! Nice catch

**Comment by GrayJack** [comment]
Fixed on commit ad98317be6a061fe6f3027116f0fc88f5676c7b6
